src/ui/server/controllers/insurance/eligibility/eligible-to-apply-online/index.test.ts

```diff
@@ -30,10 +38,52 @@ describe('controllers/insurance/eligibility/eligible-to-apply-online', () => {
   });
 
   describe('post', () => {
-    it(`should redirect to ${ROUTES.INSURANCE.ELIGIBILITY.ALREADY_HAVE_ACCOUNT}`, () => {
```

Comment:
can we dereference these?

Reply:
This has been removed? It's now `${ROOT}/${mockReferenceNumber}${INSURANCE_ROUTES.ALL_SECTIONS}`

Reply:
ignore me, my bad